requirements.dev.txt

```diff
@@ -0,0 +1,5 @@
+ansible==4.6.0
```

Comment:
Good idea!
Most probably, we could even have a `prod` requirements file, that is used in runner container creation.

Reply:
yeah, I thought about this! Probably in another PR though.